components/05-mis/03-interpolated-defaults.tf

```
@@ -22,3 +26,8 @@ data "azurerm_key_vault_secret" "kubernetes_cluster_client_id" {
   name         = "sp-object-id"
   key_vault_id = data.azurerm_key_vault.hmcts_access_vault.id
 }
+
+data "azurerm_key_vault" "acmedtssdssbox_keyvault" {
```

don't include environments in resource names as this runs across environments, 
```suggestion
data "azurerm_key_vault" "acme" {
```